contents/blog/the-posthog-array-1-34-0.md

```diff
@@ -0,0 +1,111 @@
+---
+date: 2022-03-24
+title: Array 1.34.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes", "Product updates"]
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: 
+---
+
+PostHog 1.34.0 introduces...(Excerpt info here)
+
+<blockquote class='warning-note'>
+<b>IMPORTANT!</b> Before upgrading, make sure to run all async migrations on your instance. This version will not run until async migrations are completed. Please check out the <a href="/docs/self-host/configure/async-migrations/overview" target="_blank">async migrations</a> docs for details.
+</blockquote>
+
+## PostHog 1.34.0 release notes
+
+> Wondering how to upgrade a self-hosted instance? Check out our [upgrade guide](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Homepage](#new-homepage)
+- [Data Management](#new-data-management)
+- [Property Group Filtering](#new-property-group-filtering)
+- [Smart Insight Naming](#improvement-smart-insight-naming)
+- [Staff Users](#new-staff-users)
+
+### New: Homepage
+
+PostHog now directs you to a default project homepage where you can pin your favorite dashboard to and easily access those insights!
+
+We also use the section below to flag premium features.
+
+> 🎁 This is a premium feature and requires a PostHog Scale or Enterprise license. [Learn more](/pricing).
+
+<br />
+
+### New: Data Management
```

Comment:
@alexkim205 can you write a brief summary on data management here? And link to your main blogpost and pictures if possible, thanks :D